src/base/Pie.hh

```diff
@@ -77,6 +76,9 @@ class PieSlice
     CELER_CONSTEXPR_FUNCTION size_type stop() const { return stop_; }
```

Comment:
It might useful to indicate what is the advantage (if any) using this spelling vs `begin/end`.

Reply:
Begin/end should be _iterator-like_ objects that dereference to get start/stop. I will expand the comment in the Pie.hh about the similarity to Range.